xml/System.ComponentModel.Design/ObjectSelectorEditor.xml

```diff
@@ -114,9 +114,8 @@
         <Parameter Name="treeView" Type="System.Windows.Forms.TreeView" Index="0" FrameworkAlternate="netcore-3.0;netcore-3.1" />
       </Parameters>
       <Docs>
-        <param name="treeView">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="treeView">The tree view control to modify.</param>
+        <summary>Modify a Windows Forms TreeView control to use the new Explorer style theme.</summary>
```

Comment:
> _the new Explorer style theme_

Do we have a link to an article pointing to the "new Explorer style theme" that we can add here?

Reply:
By new, they're talking Visa age in this comment. How about we use "Vista Explorer style theme". I'll look around, but the docs are going to be old.

Reply:
Maybe this one?? https://docs.microsoft.com/en-us/dotnet/framework/winforms/controls/creating-an-explorer-style-interface-with-the-listview-and-treeview

Reply:
@carlossanlop What is the best way to include a link in the docs?

Reply:
You can use:

```xml
<related type="Article" href="https://link-to-article.com">Title of the description</related>
```
[Examples](https://github.com/dotnet/dotnet-api-docs/search?q=extension%3Axml+related&unscoped_q=extension%3Axml+related).

Reply:
Thanks Carlos! I've updated the sentence with the link